src/Core/View.js

```diff
@@ -170,16 +170,19 @@ function _preprocessLayer(view, layer, provider) {
             }
         };
         defineLayerProperty(layer, 'opacity', 1.0, () => {
-            layer.object3d.traverse((o) => {
-                if (o.layer !== layer.id) {
-                    return;
-                }
-                changeOpacity(o);
-                // 3dtiles layers store scenes in children's content property
-                if (o.content) {
-                    o.content.traverse(changeOpacity);
-                }
-            });
+            const object3d = layer.object3d || layer.parentLayer.object3d;
```

Comment:
The problem is that my WFS layer doesn't have any object3d, every meshes are added to the globe tile layer.

Reply:
Apparently it's changing the opacity of the layer

What could be when attaching a geometric layer:

```js
var layer = new itowns.GeometryLayer('3d-tiles-discrete-lod', globeView.tileLayer.object3d);
globeView.addLayer({
    object3d: globeView.tileLayer.object3d,
}, globeView.tileLayer);
```

or in `View.prototype.addLayer`

```js
if(!layer.object3d) {
   layer.object3d = parentLayer.object3d
}
```

Reply:
Oh I see. I don't have anything better to propose for now, so let's keep it like that.

Reply:
But for this reason, that's why I think we should work more with groups, because most of the time, we change opacity/wireframe stuff and not layer.object3d transformations.

Reply:
actually: this code works with the assumption that when `layer.object3d` is null, this layer only attach things to parentLayer.object3d. This is not true in the general case. Therefore, we should not default to `parentLayer.object3d` here, but to `view.scene` instead.

AND we should also do, in WFS_Provider.preprocessDataLayer:
```js
layer.object3d = layer.object3d || parentLayer.object3d;
```

because it's WFS_Processing that add the resulted WFS object to the current node. The fact that the layer object3d is the same as its parent object3d is only true for WFS layers, so it makes sense that WFS preprocessing handle that. (you'll have to move `layer.parentLayer = parentLayer` in View.prototype.addLayer before preprocessing for this to work)

With this 2 changes:
1. WFS layers would be correctly configured (preprocessing phase will correctly init layers for the update method of WFS)
1. the opacity property will work in all cases

N°1 is necessary for this PR, 2 is not (but as you're going to touch this line anyway...)